### ISSUE TITLE
Cannot reproduce the RESULTS(wer:43.25%) of S5b_track1 in Chime6

### ISSUE THREAD

**Issue by dospeech** [kaldi-asr/kaldi#4701]
When I run the run.sh baseline of S5b_track1 in Chime6, I can't get the wer:43.25% of RESULTS,and I get the best result is wer:45.48% .
wer-details:
%WER 45.48 [ 26779 / 58881, 2332 ins, 11138 del, 13309 sub ] exp/chain_train_worn_simu_u400k_cleaned_rvb/tdnn1b_cnn_sp/decode_dev_gss_multiarray_2stage/wer_9_0.0

is there something wrong with my result？

**Comment by danpovey** [comment]
@desh2608 might be able to help with this.

**Comment by dospeech** [comment]
> It's hard to say what may be going wrong just based on WER. Did you change any hyperparameters? Did you look at the intermediate results (e.g. from GMM decoding) to see if they match those in the recipe?

THE wer of GMM decoding (tri3 )is 83.75%(baseline is 85.72 %), I just changed the number of gpu (gpu_num=4）and didn't change any hyperparameters.  

Whether the result of baseline (wer:43.25%) is generated by using RNNLM-rescore？

**Comment by jtrmal** [comment]
just a note -- it might be just a difference caused by different random
initialization.
y.

**Comment by dospeech** [comment]
> I think so, yeah. The WER without RNNLM rescoring should be closer to 46%. See this at the top of the run_cnn_tdnn_1b.sh: %WER 46.07 [ 27124 / 58881, 2905 ins, 9682 del, 14537 sub ] exp/chain_train_worn_simu_u400k_cleaned_rvb/tdnn1b_cnn_sp/decode_dev_gss_multiarray_2stage/wer_10_0.0
> 
> Could you try with the RNNLM rescoring?

 I‘ve tried with the  RNNLM rescoring, and the WER is 43.72%,it looks closer to 43.25(baseline).
 Is there any other optimizations?

**Comment by desh2608** [comment]
You can try tuning some of the hyperparameters (esp. learning rate) since you changed the number of training jobs (GPUs). But I think at this point you're close enough that the difference is statistically insignificant.

**Comment by dospeech** [comment]
> You can try tuning some of the hyperparameters (esp. learning rate) since you changed the number of training jobs (GPUs). But I think at this point you're close enough that the difference is statistically insignificant.

OK,thank you for your answer. Besides,is there the WER ( baseline )on the evaluation set ?
